src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -111,16 +111,8 @@ public static string BuildOperationName(IEnvironment env)
 
         internal override int Execute(IServiceProvider serviceProvider, IConsole console)
         {
-            return Execute(serviceProvider, console, stdOutHandler: null, stdErrHandler: null);
-        }
-
-        // To enable unit testing
-        internal int Execute(
-            IServiceProvider serviceProvider,
-            IConsole console,
-            DataReceivedEventHandler stdOutHandler,
-            DataReceivedEventHandler stdErrHandler)
-        {
+            DataReceivedEventHandler stdOutHandler = null;
+            DataReceivedEventHandler stdErrHandler = null;
```

Comment:
do we need these two vars? we can just pass nulls to `serviceProvider.GetRequiredService<IScriptExecutor>().ExecuteScript(...)`

Reply:
You are right, I see them only used once inside ExecuteScript(). Will remove them